ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/page/customheaderlibs.html

```diff
@@ -11,5 +11,5 @@
 <!--/* Include Context Hub */-->
 <!--<sly data-sly-resource="${'contexthub' @ resourceType='granite/contexthub/components/contexthub'}"/> -->
 <span class ="checkmode" checkmode = "${wcmmode.edit ? 'edit' : 'notEdit'}" />
-<input type = "hidden" id="isloginRequired" value= "${pageProperties.isLoginRequired}" />
```

Comment:
@DeepaliAgrwl While redirecting user on login page, keep current page path in url in query parameter so that after successful login user redirect back on previous page.